crates/nu-command/src/network/http/delete.rs

```diff
@@ -43,8 +43,8 @@ impl Command for SubCommand {
             )
             .named(
                 "max-time",
-                SyntaxShape::Int,
-                "timeout period in seconds",
+                SyntaxShape::Duration,
+                "timeout period with a duration unit (9sec)",
```

Comment:
Maybe something like this is slightly better?
```suggestion
                "max duration before timeout occurs",
```